### PR TITLE
feat: update default nodejs version to 16

### PR DESCRIPTION
**Pull request by biruwon**
🎉 Thanks for submitting a pull request! 🎉

#### Summary

As commented by @Skn0tt, we are updating the default nodejs runtime to version 16 but was missing here. The version will be updated on Monday, so we can release it after that. Let me know if I am doing something wrong as it is my first PR here 😄 

---

For us to review and ship your PR efficiently, please perform the following steps:

- [ ] Open a [bug/issue](https://github.com/netlify/zip-it-and-ship-it/issues/new/choose) before writing your code 🧑‍💻.
      This ensures we can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing
      a typo or something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
- [ ] Read the [contribution guidelines](../CONTRIBUTING.md) 📖. This ensures your code follows our style guide and
      passes our tests.
- [ ] Update or add tests (if any source code was changed or added) 🧪
- [ ] Update or add documentation (if features were changed or added) 📝
- [ ] Make sure the status checks below are successful ✅

**A picture of a cute animal (not mandatory, but encouraged)**

### CODE REVIEW

File: tests/unit/runtimes/node/utils/node_version.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ test('getNodeVersion', (t) => {
   t.is(getNodeVersion('nodejs8.x'), 8)
   t.is(getNodeVersion('12.x'), 12)
   t.is(getNodeVersion('8.x'), 8)
-  t.is(getNodeVersion('node14'), DEFAULT_NODE_VERSION)
+  t.is(getNodeVersion('node16'), DEFAULT_NODE_VERSION)
```
Comment (danez) [inline tests/unit/runtimes/node/utils/node_version.js:14]: Can we add a line here to check that 14.x still works.

**Review by danez — DISMISSED** [review]
I guess we need to coordinate this release with bitballon exactly?

File: tests/unit/runtimes/node/utils/node_version.js (hunk 1 above)
Comment (biruwon) [reply tests/unit/runtimes/node/utils/node_version.js:14]: Ok, I will take advantage of the problems reported by the bots to add this

Comment (biruwon) [reply tests/unit/runtimes/node/utils/node_version.js:15]: Solved [here](https://github.com/netlify/zip-it-and-ship-it/pull/1138/commits/27835289778d5f3a4aff61b332ec986c4df69b18)

**Comment by biruwon** [comment]
BTW @danez, I saw the bot reported [like a 50% decreased ](https://github.com/netlify/zip-it-and-ship-it/pull/1138#issuecomment-1178771955). I guess that good, but not sure if is it related to the nodejs version or just some random numbers that depends on the time it was executed 🤔

**Comment by biruwon** [comment]
> BTW @danez, I saw the bot reported [like a 50% decreased ](https://github.com/netlify/zip-it-and-ship-it/pull/1138#issuecomment-1178771955). I guess that good, but not sure if is it related to the nodejs version or just some random numbers that depends on the time it was executed 🤔

I just did another commit and results are totally different, so already answered 😄

File: tests/unit/runtimes/node/utils/node_version.js [hunk 2]
```diff
@@ -7,11 +7,12 @@ const {
 } = require('../../../../../dist/runtimes/node/utils/node_version.js')
 
 test('getNodeVersion', (t) => {
+  t.is(getNodeVersion('nodejs14.x'), 14)
   t.is(getNodeVersion('nodejs12.x'), 12)
   t.is(getNodeVersion('nodejs8.x'), 8)
   t.is(getNodeVersion('12.x'), 12)
   t.is(getNodeVersion('8.x'), 8)
-  t.is(getNodeVersion('node14'), DEFAULT_NODE_VERSION)
+  t.is(getNodeVersion('node16'), DEFAULT_NODE_VERSION)
```
Comment (danez) [inline tests/unit/runtimes/node/utils/node_version.js:15]:
```suggestion
  t.is(getNodeVersion('16.x'), DEFAULT_NODE_VERSION)
```

**Comment by danez** [comment]
Just too clarify again, this has nothing todo with the runtime, but with the output target of our bundlers. So with the runtime now being default nodejs 16 in bitballon, but we still bundle the functions here for node14 (which is fine and works, but can be better optimized if we also set it to node16). This PR changes that.

File: tests/unit/runtimes/node/utils/node_version.js (hunk 2 above)
Comment (biruwon) [reply tests/unit/runtimes/node/utils/node_version.js:15]: I think this unit test check that something like `"nodeVersion"` is invalid so it uses the default one. Not sure if we should change it

**Review by danez — APPROVED** [review]

**Pull request merged**